engine/src/main/java/org/terasology/entitySystem/prefab/internal/PrefabFormat.java

```diff
@@ -54,7 +54,8 @@ public PrefabData load(ResourceUrn resourceUrn, List<AssetDataFile> inputs) thro
                 PrefabSerializer serializer = new PrefabSerializer(componentLibrary, typeSerializationLibrary);
                 return serializer.deserialize(prefabData);
             } else {
-                throw new IOException("Failed to read prefab for '" + resourceUrn + "'");
+                logger.error("Failed to read prefab for '" + resourceUrn + "'");
```

Comment:
This method was designed to throw a IOException in case something went wrong (this is reflected in the signature). Imho we should keep this design, catching (and handling) errors too early can be problematic.